### DISCUSSION TITLE
Merge selected multiple line from left or right side

### DISCUSSION THREAD

**Discussion by bobydo** [WinMerge/winmerge#2347] [Ideas]
Hello,
I have files that include design code  at top (E.g. line 1 to 500) and VB code at bottom (E.g. line 501 to 700). Most time, I like to merge left side changes to right side change for design code (line 1 to 500) but keep VB code  (line 501 to 700), using Alt + down and Alt + right is very slow. It there a away to select all design code  and use Alt + right to merge left changes to right side? thanks. Shenyi

**Comment by sdottaka** [comment]
If you select multiple lines as shown below and choose the Copy to Right menu item or press Alt+Right key, multiple Diff hunks in the selection will be copied to the right.

![image](https://github.com/WinMerge/winmerge/assets/98126/2e1a0c9a-c87c-4f95-9610-ec19cf430799)
